### ISSUE TITLE
Navbar Cursor control keys with Swipe Up gesture

### ISSUE THREAD

**Issue by creeve4** [GravityBox/GravityBox#1996]
When using Navbar Cursor control keys with Swipe Up gesture, the back button is moved very close to the home button (pill) when the cursor keys are shown. This is on my Nexus 5X.

![Screenshot_20190615-111405](https://user-images.githubusercontent.com/1910130/59554383-3972ca00-8f5f-11e9-846a-5e105980ebcb.png)

**Comment by C3C0** [comment]
Yeah, I had a hard time adjusting support for gestured navbar ( https://github.com/GravityBox/GravityBox/commit/4f2529814cb9c77cab895e96d714c7e9ad9fed66 ) and currently it's the best what I could come up with. My possibilities using Xposed are limited.

**Comment by creeve4** [comment]
I did find that if I use a compact navbar layout (which I prefer anyway), I
don't have this problem.
